core/src/main/java/io/questdb/griffin/SqlCodeGenerator.java

```diff
@@ -3482,6 +3482,18 @@ private RecordCursorFactory generateOrderBy(
                     final CharSequence column = orderByColumnNames.getQuick(i);
                     int index = metadata.getColumnIndexQuiet(column);
 
+                    if (index < 0) {
+                        ObjList<ExpressionNode> nodes = model.getOrderBy();
+                        int position = 0;
+                        for (int j = 0, y = nodes.size(); j < y; j++) {
```

Comment:
loop has `'j` as the index, loop body uses `i` - tests pass - please increase test coverage, consider fuzzing

Reply:
same issue in the loop below